2019/lib/mse/msutil.js

```diff
@@ -781,15 +781,44 @@ window.setupMse = function(video, runner, videoStreams, audioStreams) {
       var audioStream = audioStreams[audioStreamIdx];
       if (audioStream != null) {
         audioSbs.push(ms.addSourceBuffer(audioStream.mimetype));
-        appendLoop(audioStream, audioSbs[audioSbs.length - 1]);
+        if (harnessConfig.testTime > 15) {
```

Comment:
Would you mind to chime in a developer from Google here? It would be good to get some clarification, and I believe that this is also a feature they would like to have.

Reply:
That would be great! Would you know of anyone that could help with this?

Reply:
Please check the original repository. Pick who mainly lands code changes?